scripts/etc/init.d/rcS

```diff
@@ -838,29 +851,33 @@ echo "OK"
 # networking
 echo -n "  Configuring network settings... "
 touch /rootfs/etc/network/interfaces || fail
-{
-    # wheezy seems to add the lo interface, so first check for it
-    if ! grep -q "auto lo" /rootfs/etc/network/interfaces; then
-        echo "auto lo"
-        echo "iface lo inet loopback"
-        echo ""
-    fi
+# lo interface may already be there, so first check for it
+if ! grep -q "auto lo" /rootfs/etc/network/interfaces; then
+    echo "auto lo" >> /rootfs/etc/network/interfaces
+    echo "iface lo inet loopback" >> /rootfs/etc/network/interfaces
+fi
 
-    # eth0 config
-    echo "auto $ifname"
-    if [ "$ip_addr" = "dhcp" ]; then
-        echo "iface $ifname inet dhcp"
-    else
-        echo "iface $ifname inet static"
-        echo "    address $ip_addr"
-        echo "    netmask $ip_netmask"
-        echo "    broadcast $ip_broadcast"
-        echo "    gateway $ip_gateway"
-    fi
-    if [ "$ifname" != "eth0" ]; then
-        echo "wpa-conf /boot/config/wpa_supplicant.conf"
+# eth0 config
+echo "" >> /rootfs/etc/network/interfaces
+echo "auto $ifname" >> /rootfs/etc/network/interfaces
```

Comment:
With wireless interfaces, you usually also want `allow-hotplug $ifname` as wireless devices are more likely to get unplugged and replugged, but also it will give a signal when the device is ready to be used by which the `auto` may already have passed.